lxd/loki/loki.go

```diff
@@ -330,14 +330,22 @@ func (c *Client) HandleEvent(event api.Event) {
 			}
 		}
 
-		messagePrefix := ""
+		keys := make([]string, len(context))
```

Comment:
This should be `make([]string,0, len(context))` so the capacity is pre-initialized but not the elements themselves so that when its appended to below it wont have several empty elements already added.